AddOns/BasicFinancials/app/src/codeunit/AssistedSetupBF.Codeunit.al

```diff
@@ -0,0 +1,46 @@
+codeunit 20605 "Assisted Setup BF"
+{
+    Access = Internal;
+
+    var
+        AssistedSetupTxt: Label 'Set up the Business Central Basic extension';
+        AssistedSetupHelpTxt: Label 'https://go.microsoft.com/fwlink/?linkid=', Locked = true;
+        AssistedSetupDescriptionTxt: Label 'Business Central Basic';
```

Comment:
I think we need some better description here. 
![image](https://user-images.githubusercontent.com/34042170/85368669-fbbe5f00-b52b-11ea-9d03-9321823454c2.png)

Reply:
There is actual no setup in the the assisted Setup, it make no different if it is completed or not. It is just some sort of information. I don't know have to come up with a better description.

Reply:
Disagree here. So, even we have only one page, it is still some kind of setup. Additionally, the extension will be present for all users, so we should provide proper description what is it used for.

Reply:
Yep, sorry this part is not for you. Will talk to Brent.